gulpfile.mjs

```diff
@@ -583,6 +584,24 @@ function createImageDecodersBundle(defines) {
     .pipe(tweakWebpackOutput("pdfjsImageDecoders"));
 }
 
+function createFluentBundle(defines) {
```

Comment:
Why do we need this, can't we "simply" leverage import maps *fully* to pull in the relevant code directly in the viewer?

Ideally the relevant build-targets would automatically include the necessary Fluent-code during building.